### DISCUSSION TITLE
Ecommerance Plugins (Opencart, Prestashop, Woocommerance)

### DISCUSSION THREAD

**Discussion by trymeouteh** [MrNaif2018/bitcart#284] [Ideas]
Opencart, Prestashop and Woocommerance are common open source ecommerance platforms and a easy to install plugin to use bitcartcc with these platforms would help self hosted FOSS stores online accept crypto without using custody services.

**Comment by MrNaif2018** [comment]
We have whmcs and woocommerce plugins. Is opencart that popular actually? I think I would also add shopify plugin

**Reply by trymeouteh** [reply]
I do think opencart is getting more popular. I plan on learning it after I learn wordpress.

**Reply by MrNaif2018** [reply]
I see. I will check it, but if it is done I will need your help with testing. Are you in any of our community groups?

**Reply by trymeouteh** [reply]
I do not have telegram or discord. I will be willing to help test these plugins once I learn how to create a opencart and prestashop website.
